test/postman/BCParis_Env.postman_environment.json

```diff
@@ -0,0 +1,29 @@
+{
+	"id": "faddc1ec-d7aa-4e77-a4ec-25fdaaa8307d",
+	"name": "BCParis_Env",
+	"values": [
+		{
+			"key": "dev_username",
```

Comment:
usually you let the user create multiple env, so should be `username` and then I can create as many env that I want on postman and switch